### ISSUE TITLE
Rotating verb needs a cooldown

### ISSUE THREAD

**Issue by DogZeroX** [space-wizards/space-station-14#5171]
It can be abused with an autoclicker
![image](https://user-images.githubusercontent.com/43253663/140457687-0bdbe1fb-515e-4621-b3bf-649b45ec4952.png)

https://user-images.githubusercontent.com/43253663/140457739-8567504f-8630-4a21-801a-99b173a18217.mp4

**Comment by mirrorcult** [comment]
should verbs in general have a cooldown?

**Comment by Ephememory** [comment]
they should definitely have the option for it

**Comment by SweptWasTaken** [comment]
> 
> 
> should verbs in general have a cooldown?

Yes. You'd have to do some useless stuff to [RotateableSystem.cs] to actually give it cooldowns atm would be much better to just add a `Verb.Cooldown = 100`
